lib/gollum/views/compare.rb

```diff
@@ -19,11 +19,13 @@ def after
 
       def lines
         lines = []
-        @diff.split("\n")[2..-1].each_with_index do |line, line_index|
+        lines_to_parse = @diff.split("\n")[2..-1]
+        lines_to_parse = lines_to_parse[1..-1] if lines_to_parse[0].start_with?('+++')
```

Comment:
Why was this `starts_with('+++')` introduced?

Reply:
Allowing diffing of files with different filenames in different revisions (which is the main purpose of this PR) introduced a new edge case: the patch 'header' would be one line longer to indicate the creation of a 'new' file. That header line starts with `+++`. We want to start parsing changes from the next line.

Reply:
Good. Could you add a comment above this line to express this (e.g., `# skip optional creation header` or something)?